### PR TITLE
Update ESmy012.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
new ms. I still did not type in incipits for Commemorative notes and Salams;  maybe Ethiopic in those parts is dispensible? The description of this recent, not very important  ms will be complex and occupy a lot of space.

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
COuld you please lok through, comment/merge?

File: ES/ESmy012.xml [hunk 1]
```diff
@@ -38,118 +38,573 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                              </msItem>
+                                              <msItem xml:id="ms_i1.13.4">
+                                                 <locus target="#99va"/>
+                                                 <title type="complete" ref="LIT1295Dersan#SalamTeqemt"/>
+                                            <textLang mainLang="gez"/>
+                                            </msItem>
                                                   </msItem>
                                                   </msItem>
 
+
```
Comment (eu-genia) [inline ES/ESmy012.xml:592]: add closing tag

File: ES/ESmy012.xml [hunk 2]
```diff
@@ -38,118 +38,573 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                                    በከመ፡ ታለምድ፡ ዘልፈ፨ ለረዲኦትየ፡ ነዓ፡ ሰፈሐከ፡ ክንፈ፨
                                                      </incipit>
-                     <incipit>
-                        <foreign xml:lang="en"/>
-                     </incipit>
-                      <note/>
-                  </msItem>
+                                                     </msItem>
+                                                     </msItem>
+
```
Comment (eu-genia) [inline ES/ESmy012.xml:606]: remove closing tag, to fix nesting

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy012.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy012.xml:592]: ok

File: ES/ESmy012.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy012.xml:606]: ok

**Review by eu-genia — APPROVED** [review]

**Pull request merged**